core/CMakeLists.txt

```diff
@@ -5,6 +5,13 @@ else(APPLE)
 	set(CORE_EXTRA_SRCS "")
 endif(APPLE)
 
+if(NOT DEFINED WITH_SUPERTIMESTREAM)
+	set(WITH_SUPERTIMESTREAM TRUE CACHE BOOL "Build G3SuperTimestream class")
+endif()
+if(WITH_SUPERTIMESTREAM)
+	set(CORE_EXTRA_SRCS ${CORE_EXTRA_SRCS} src/G3SuperTimestream.cxx)
+endif()
+
```

Comment:
Any reason to make this conditional?

Reply:
Largely for ease of transition on the SO side, I think.

Reply:
Maybe we can invert the sense of the flag then? Have a "don't build this" flag instead of a "build this" one?

Reply:
Done -- `WITHOUT_SUPERTIMESTREAM`